### PR TITLE
Data updates

### PR DESCRIPTION
**Pull request by jlhogan**
- exposes Language Spoken on CareProvider type as `languages`
- updates all string list types to be boolean maps with explicit typing

### CODE REVIEW

File: src/types.ts [hunk 1]
```diff
@@ -18,6 +18,78 @@ export type WeeklyHours = {
...
+
+export const FEES = [
+  "Medicaid",
+  "PrivateInsurance",
+  "SelfPay",
+  "SlidingFeeScale",
+] as const;
+export type Fees = typeof FEES[number];
+
+export const LANGUAGES = ["English", "Spanish", "Vietnamese"] as const;
```
Comment (cathydeng) [inline src/types.ts:90]: i saw a bunch of other languages in the ladders extract (cantonese, mandarin, russian, etc) - would be nice for this script to print out a warning if the export has items in a list that we don't expect

**Review by cathydeng — APPROVED** [review]
🙏

File: src/translations/en/components.json [hunk 2]
```diff
@@ -11,6 +11,12 @@
     "website": "Website",
     "address": "Address",
     "fees": "Fees",
+    "_fees": {
+      "Medicaid": "Medicade",
+      "PrivateInsurance": "Private Insurance",
+      "SelfPay": "Self Pay",
```
[the remark this answers is not in the document]
Comment (jlhogan) [reply src/translations/en/components.json:17]: yeah i just left this all as it was coming out of the ladders extract, but we can definitely change the display translations -- I'd agree that sentence case is preferred so i'll just take our agreement as go ahead to make that change. RE hyphenation, I'll leave it for now to match ladders data but we can follow up with CO folks

**Review by cathydeng — APPROVED** [review]
🔑

**Pull request merged**